homeassistant/helpers/entity.py

```diff
@@ -388,6 +388,9 @@ def async_registry_updated(self, old, new):
 
         self.hass.async_create_task(readd())
 
+    async def async_added_to_hass(self: 'Entity') -> None:
```

Comment:
This will make adding each entity slower, as it does an extra, unnecessary yield to the event loop.

Reply:
Also no need to type `self`.

Reply:
Is that true? I couldn't find a definitive answer, but it seems that when calling `await` still enters the next function until it hits a coroutine that doesn't immediately return.